openapi2beans/JavaChecker/src/main/java/dev/galasa/openapi2beans/example/generated/BeanWith3DNestedArray.java

```diff
@@ -0,0 +1,22 @@
+package dev.galasa.openapi2beans.example.generated;
+
+// A bean with a nested 3d array
+public class BeanWith3DNestedArray {
+    // Class Variables //
+    private String[][][] anArrayVariable;
```

Comment:
an array of array of array of String ?
Have you reviewed this PR before I got here ? Reviewing your own work is almost always going to find some of the issues a reviewer would otherwise find, before they find them.